test/testdata/prometheus-metric-tests.yaml

```diff
@@ -0,0 +1,90 @@
+- name: up
```

Comment:
Consider adding a comment to the top of this file which provides some context about it's existence, how it is called/used, e.t.c. (maybe a link to KUDO) as that can make it easier for future readers.

Reply:
good idea. done.